### DISCUSSION TITLE
Default userID imported through relation from an auth collection

### DISCUSSION THREAD

**Discussion by jeichel09** [pocketbase/pocketbase#5311] [Q&A]
Hello friends,

i have a quick question. I have two collections in my latest project - users and transactions. The user-collection is a Auth one and the other is a base one. I wanted to import the unique and randomly created by Pocketbase user id from the users-collection through relation into the transactions-collection. My code in the app then did what it did. It made records of a couple of transactions. But when I 
 looked into the admin panel to see the records, the userID was actually the user's email and not it's id. Is there a way (may be using some API rule) to change this behavior, so that i get the unique id instead?
 Thanks in advance for any ideas!
 Daniel

**Comment by ganigeorgiev** [comment]
You'll have a better chance someone to assist you if you export your collections configuration as json (_Admin UI > Settings > Export collections_), provide a code sample with your import script and elaborate more clearly on the actual problem because both the title and the description are confusing (at least to me).

**Reply by ganigeorgiev** [reply]
If you mean that the user `relation` field is displaying the related user email instead of its id in the Admin UI, then this is because of its default representation of the `relation` field. The actual value in the API response should be the related user id. At the moment it is not possible to force displaying the id in the Admin UI, but with the [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355) it will be supported since the `id` field will be a regular collection schema field and you'll be able to mark it "Presentable" to overwrite the default Admin UI behavior.

**Comment by jeichel09** [comment]
[pb_schema.json](https://github.com/user-attachments/files/16492448/pb_schema.json)
